src/js/utils.js

```diff
@@ -107,8 +107,13 @@ this.exports = {
   },
   
   hasSensors: function(thermostat) {
-    return true; //TODO: Fixme!
+    var hasSensors = false;
+    thermostat.remoteSensors.forEach(
+      function(sensor) {
+          if (sensor.type ==='ecobee3_remote_sensor') {
```

Comment:
I think it'll be just fine to have a condition to check if remote sensors array size is greater then zero. This is remote sensors, so thermostat-internal sensors won't be here at all. I bet flair sensors won't be in the list, but even if they are - it doesn't hurt to have them displayed on Pebble.